src/docbuilder/rustwide_builder.rs

```diff
@@ -589,6 +579,12 @@ impl RustwideBuilder {
 
         // Add docs.rs specific arguments
         let mut cargo_args = Vec::new();
+        // Show the command line Rustdoc is invoked with
+        cargo_args.push("--verbose".into());
+        // We know that `metadata` unconditionally passes `-Z rustdoc-map`.
+        // Don't copy paste this, since that fact is not stable and may change in the future.
+        // Normally we would need -Z unstable-options, but we currently pass that too.
```

Comment:
We didn't in my testing, I think it's only when the crate has `package.metadata.docs.rs.rustc-args` set that we do.